src/components/popup-dialog/PopupDialog.styles.ts

```diff
@@ -12,5 +12,43 @@ export const styles = {
     position: 'absolute',
     right: { xs: '8px', sm: '20px' },
     top: { xs: '8px', sm: '20px' }
+  },
+  dialogPaper: {
+    padding: '20px',
+    borderRadius: '8px'
+  },
+  dialogTitle: {
+    fontWeight: 'bold',
+    color: '#566970'
```

Comment:
Regarding the color hex in the css files. We shouldn't create hex each time and duplicate it. The best practice is use variables that are already set in src/styles/app-theme/app.pallete.ts. file. If you take this hex from mockups and in the variables no this color yet please add this hex color to relevant basic, error, warning,... section.

Reply:
I second, fixed it